.github/workflows/release.yml

```diff
@@ -6,65 +6,65 @@ on:
     paths:
       - '.github/project.yml'
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash
+
 jobs:
   release:
     runs-on: ubuntu-latest
     name: release
     if: ${{github.event.pull_request.merged == true}}
 
     steps:
-      - uses: radcortez/project-metadata-action@master
-        name: retrieve project metadata
+      - uses: radcortez/project-metadata-action@main
+        name: Retrieve project metadata
         id: metadata
         with:
           github-token: ${{secrets.GITHUB_TOKEN}}
           metadata-file-path: '.github/project.yml'
 
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
 
-      - name: Import GPG key
-        id: import_gpg
-        uses: crazy-max/ghaction-import-gpg@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v4
         with:
+          distribution: temurin
+          java-version: 11
+          cache: 'maven'
+          server-id: ossrh
+          server-username: MAVEN_USERNAME
```

Comment:
Where is it coming from?

Reply:
These are the environment properties set in the generated settings.xml during the release. They are set in https://github.com/quarkiverse/quarkus-mybatis/pull/403/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R62